src/commcare_cloud/ansible/roles/elasticsearch/tasks/misc_v2.yml

```diff
@@ -0,0 +1,38 @@
+
+- name: Copy Elasticsearch logging config
+  become: yes
+  template:
+    src: config/logging.yml.j2
+    dest: "{{ elasticsearch_conf_dir }}/logging.yml"
+    owner: elasticsearch
+    group: elasticsearch
+  register: copy_elasticsearch_logging_result
+  tags:
+    - es_conf
+
+- name: Disable Java Heap Dump
+  replace:
+    path: "{{ elasticsearch_home }}/bin/elasticsearch.in.sh"
+    regexp: '^JAVA_OPTS="\$JAVA_OPTS -XX:\+HeapDumpOnOutOfMemoryError"'
+    replace: '#JAVA_OPTS="$JAVA_OPTS -XX:+HeapDumpOnOutOfMemoryError"'
+
+- name: Install elasticsearch python client
+  become: yes
+  pip:
+    name: elasticsearch>=2.0.0,<3.0.0
+  when: elasticsearch_version == '2.4.6'
```

Comment:
seems unnecessary since the whole file is version scoped

Reply:
Sure